rest-api/pom.xml

```diff
@@ -38,10 +38,19 @@
 
     <dependencies>
         <!-- Project dependencies -->
+        <dependency>
+            <groupId>org.jboss.pnc</groupId>
+            <artifactId>pnc-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.jboss.pnc</groupId>
             <artifactId>dto</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <scope>provided</scope>
```

Comment:
It's used in tests only? If so, please change it to test scope

Reply:
It's the same as we have in the DTO module.

Reply:
but in DTO it¨s because of org.hibernate.validator.constraints.URL that is used in DeliverablesAnalysisRequest

Reply:
ok, changing it to test.